### ISSUE TITLE
[Problem/Bug]: WebView2 Fixed Runtime does not support PlayReady functionality.

### ISSUE THREAD

**Issue by cryptofyre** [MicrosoftEdge/WebView2Feedback#4632]
### What happened?

We're encountering an issue where the WebView2 Fixed Runtime does not support PlayReady functionality. Applications such as ours (Cider) that rely on PlayReady DRM for media playback experience compatibility problems when using the WebView2 Fixed Runtime. This lack of support results in failures to play protected content as expected.

Currently, this limitation is impacting our ability to provide secure media playback within our application. We need PlayReady support in the WebView2 Fixed Runtime to ensure compatibility with our DRM requirements and maintain a consistent user experience.

### Importance

Blocking. My app's basic functions are not working due to this issue.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

123.0.2420.81

### SDK Version

1.0.1264.42

### Framework

WPF

### Operating System

Windows 11

### OS Version

26100.560

### Repro steps

1. **Set up the Environment**
   - Create a simple basic WPF WebView2 project.
   - Setup the latest version of the WebView2 Fixed Runtime instead of using Evergreen/System's installation.

3. **Prepare project**
   - Implement or test content protected by PlayReady DRM in the application.

4. **Launch the Application**
   - Start the application and navigate to a page or content that requires PlayReady DRM for playback.

5. **Observe the Issue**
   - Attempt to play the DRM-protected content.
   - Application does not properly playback the encrypted stream/video/audio.

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

Don't know

**Comment by victorhuangwq** [comment]
By specifically mentioning that the Fixed Runtime does not support PlayReady, are you implying that the Evergreen WebView2 runtime supports PlayReady?

**Comment by cryptofyre** [comment]
> By specifically mentioning that the Fixed Runtime does not support PlayReady, are you implying that the Evergreen WebView2 runtime supports PlayReady?

This is correct yes, we are able to use PlayReady implementation just fine if we select the Evergreen runtime instead.
